docs/helpers.md

```diff
@@ -36,6 +36,23 @@ let data = (await $axios.get('...')).data
 let data = await $axios.$get('...')
 ```
 
+### `setBaseURL(baseURL, browserBaseURL = null)`
+
+Axios instance has a helper to easily set baseURL or browserBaseURL.
```

Comment:
I would suggest to briefly explain use case. Like do use it when you need a dynamic runtime url. Otherwise use config and env variables (or server it can be even runtime)